tools/publish-lib.js

```diff
@@ -71,8 +71,8 @@ async function main() {
   await writeLib('main', `${lib}-nightly`);
   console.log('  done');
 
-  console.log('Update README and LICENSE if needed...');
-  for (const file of ['README.md', 'LICENSE']) {
+  console.log('Update README, LICENSE and demo.html if needed...');
+  for (const file of ['README.md', 'LICENSE, demo.html']) {
```

Comment:
```suggestion
  for (const file of ['README.md', 'LICENSE', 'demo.html']) {
```
(but `demo.html` does not exist in the root folder in any case, so that won't be enough)